modules/nixos.nix

```diff
@@ -34,29 +34,127 @@ in
             default.value = [ ];
           };
 
+          home-manager = lib.options.create {
+            description = "The home-manager input to use.";
+            type = lib.types.raw;
+            default.value =
+              if inputs ? home-manager
+              then inputs.home-manager.result
+              else null;
+          };
+
+          homes = lib.options.create {
+            description = "A list of homes to activate for the system.";
+            type = lib.types.list.of lib.types.string;
```

Comment:
Any thoughts on this being a set of portable home definitions? Seems like that would make things more flexible when attempting to extend/override.